PowerPointLabs/PowerPointLabs/TimerLab/TimerLabPaneWPF.xaml.cs

```diff
@@ -87,10 +97,10 @@ private int LineMarkerColor()
         #endregion
 
         #region Timer Creation
-        private void CreateBlocksTimer(int duration, float timerWidth, float timerHeight, float timerLeft, float timerTop)
+        private void CreateBlocksTimer(int duration, float timerWidth, float timerHeight, float timerLeft, float timerTop, bool isCountdown)
```

Comment:
Instead of using a boolean and a checkbox for isCountdown, would it be nicer to have a dropdown box and allowing the user to select the type of timer that he/she wants? Thus it will be an enum for the type of timer rather than having a boolean for isCountdown. This way, in the future we might have other types of timers...? Or is there a reason why boolean is better?

Reply:
I decided to stick with a boolean for now, for simplicity. I attempted to use a `ComboBox` with 2 options and create an enum for the type of timer, however I found that it created a lot of unnecessary code and complications... Furthermore, I can't really think of any other "types" of timers except maybe a digital timer, or an analog timer, but both of which if implemented would require major changes to much of the code anyway. 

Perhaps I'm missing some other "types" of timers that you have in mind? But of course, if a dropdown menu preferred, that can be done as well!